channels/layers.py

```diff
@@ -341,16 +345,16 @@ async def group_add(self, group, channel):
         Adds the channel name to a group.
         """
         # Check the inputs
-        assert self.valid_group_name(group), "Group name not valid"
-        assert self.valid_channel_name(channel), "Channel name not valid"
+        self.require_valid_group_name(group)
+        self.valid_channel_name(channel), "Channel name not valid"
```

Comment:
Can you rename `valid_channel_name` to `require_valid_channel_name` to match?

And also remove the `, "Channel name not valid"` since it isn't used?